### PR TITLE
WIP: preact-adapter

### PR DESCRIPTION
**Pull request by flemmingmiguel**
Preact adapter for virtual, tested on a preact project 

Opening PR for others to test and to start creating some examples

### CODE REVIEW

File: packages/preact-virtual/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "engines": {
+    "node": ">=12"
+  },
+  "files": [
+    "build/**",
+    "src"
+  ],
+  "dependencies": {
+    "@tanstack/virtual-core": "3.0.0-beta.23"
+  }
```
Comment (piecyk) [inline packages/preact-virtual/package.json:41]: Let's add `peerDependencies` with preact

**Review by piecyk — COMMENTED** [review]
Looks good 👏 Please add peerDependencies, and maybe some simple tests.

**Comment by piecyk** [comment]
@flemmingmiguel also missing build configuration for preact-virtual package

**Comment by tannerlinsley** [comment]
Out of date. Please reopen.

**Pull request closed without merging**